services/gateway/api/types/org.go

```diff
@@ -32,6 +32,10 @@ type OrgResponse struct {
 	Visibility Visibility `json:"visibility,omitempty"`
 }
 
+type UpdateOrgRequest struct {
+	Visibility Visibility `json:"visibility"`
```

Comment:
Should be a pointer and should update it only if nil. See other update apis like project.